validation/src/test/scala/hmda/validation/rules/lar/quality/Q059Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q059Spec extends LarEditCheckSpec with BadValueUtils {
+  property("Lars with loan type equal to 1 must pass") {
```

Comment:
I haven't heard any such discussion, but I would also be fine with updating our approach (i.e. using the strategy Nick is using here) for Quality edits.